### PR TITLE
Created View Item fragment

### PR DESCRIPTION
**Pull request by MatthewNeufeld**
- created fragment for viewing details of an item (ViewItemFragment.java + fragment_view_item.xml)
- fragment can be accessed by clicking on viewItemButton (initialized in ItemAdapter id: _action_view_)
- sent details of item from ItemAdapter to ViewItemFragment
- added more getters for Item class

### CODE REVIEW

**Review by ldbonkowski — COMMENTED** [review]
Looks great! I was reviewing the requirements and looks like we need javadocs for everything. I'll be making a PR shortly with docs for my functions. If you could add those to all the ones your PR has created that would be great!

File: app/src/main/java/com/example/househomey/ItemAdapter.java [hunk 1]
```diff
@@ -44,6 +54,29 @@ public View getView(int position, @Nullable View convertView, @NonNull ViewGroup
...
+            Bundle details = new Bundle();
+            details.putString("make", item.getMake());
+            details.putString("model", item.getModel());
+            details.putString("serialNumber", item.getSerialNumber());
+            details.putString("cost", item.getCost().toString());
+            details.putString("comment", item.getComment());
+            viewItemFragment.setArguments(details);
+
+            // Begin a fragment transaction
+            FragmentTransaction transaction = fragmentManager.beginTransaction();
```
Comment (ldbonkowski) [inline app/src/main/java/com/example/househomey/ItemAdapter.java:71]: @owencooke Not sure if ```navigateToFragment()``` from main activity could be made static and used here and consolidated with the ```returnToHomeScreen()``` from AddItemFragment. Might be worth looking into

Comment (owencooke) [reply app/src/main/java/com/example/househomey/ItemAdapter.java:79]: Yep good idea. I can look into adding that in my next PR

File: app/src/main/res/layout/fragment_view_item.xml
Comment (owencooke) [inline app/src/main/res/layout/fragment_view_item.xml:1]: Do you think it's possible to change this from a Relative layout to a LinearLayout or something similar? 
I think the RelativeLayout with all these hardcoded DP sizes and margins might be difficult to maintain in the long run..

**Review by owencooke — CHANGES_REQUESTED** [review]
Overall functionality works great! Just a couple notes about the XML from me (sorry lmao). 
In addition to the inline reviews:
1. I think you were right, the grey color you're using for the Make, Model, etc. labels is too light. You should be able to add this to the colors folder and use it instead of the grey. I think that grey was for other components and I missed this color
`<color name="subTitle">#7A7289</color>`

2. The edit button is totally fine brown. Maybe just make it a little closer in size to the Figma design though. Right now its chonkyyy

Screenshot (652)

**Review by owencooke — APPROVED** [review]
Thanks for making those changes, new branch looks great 🚀

**Review by ldbonkowski — APPROVED** [review]
Looks good

**Pull request merged**